src/style.css

```
@@ -1 +1,123 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
+  font-weight: 800;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  padding: 0;
+  border: 0;
+  font-weight: 300;
+}
+
+input {
+  width: 100%;
+}
+
+input::placeholder {
+  color: inherit;
+}
+
+/* #region bar */
+.search-bar {
+  display: flex;
+  align-items: center;
+  flex-shrink: 0;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  margin-top: 20px;
+  position: relative;
+}
+
+.search-bar__big {
+  height: 70px;
+}
+
+.search-bar__small {
+  height: 42px;
+}
+
+.search-bar::after {
+  background-image: url(./images/Search.svg);
+  flex-shrink: 0;
+  background-size: contain;
+  content: "";
+  position: absolute;
+}
+
+.search-bar__big::after {
+  height: 19px;
+  width: 19px;
+  left: 26px;
+}
+
+.search-bar__small::after {
+  height: 11px;
+  width: 11px;
+  left: 13px;
+}
+
+.search-bar:hover {
+  background: #FFF;
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.20);
+}
+
+.search-bar:focus {
+  border-radius: 3px;
+  background: linear-gradient(180deg, #FFF 0%, #F6F6F7 100%);
+}
+
+.search-bar input[type="text"] {
+  border: none;
+  outline: none;
+}
+
+/* #endregion */
+
+/* #region text */
+.search-bar__text {
+  display: flex;
+  align-items: center;
+  flex-shrink: 0;
+  color: #3d4e61;
+  font-family: Avenir, sans-serif;
+  font-weight: 400;
+  line-height: normal;
+}
+
+.search-bar__text-big {
+  margin-left: 60px;
+  padding-right: 29px;
+  font-size: 16px;
+  width: 129px;
+  height: 22px;
+}
+
+.search-bar__text-small {
+  justify-content: flex-end;
+  font-size: 14px;
+  width: 126.29px;
+  height: 20px;
+  padding-left: 33px;
+}
```

```suggestion
.search-bar__text-small {
  justify-content: flex-end;
  font-size: 14px;
  height: 20px;
  padding-left: 33px;
}
```
do not hardcode width. it will make visible placeholder and will be possible to write text on full input

---

Thank you for comments! I've changed my code